src/app/modules/onboarding/onboarding.component.html

```diff
@@ -3,14 +3,14 @@
     <div *ngSwitchCase="States.PickingWallet">
       <app-choose-wallet-kind
         [walletSelections]="walletSelections"
-        [selectedWallet$]="selectedWallet$"
-      ></app-choose-wallet-kind>
+        [selectedWallet$]="selectedWallet$"></app-choose-wallet-kind>
     </div>
     <div *ngSwitchCase="States.ImportWizard">
       <app-nonhd-wallet-wizard [resetOnboarding]="resetOnboarding.bind(this)"></app-nonhd-wallet-wizard>
     </div>
     <div *ngSwitchCase="States.RecoverWizard">
-      <app-hd-wallet-wizard [resetOnboarding]="resetOnboarding.bind(this)"></app-hd-wallet-wizard>
+      <app-wallet-recover-wizard (onBackToWalletsRaised)="resetOnboarding()"></app-wallet-recover-wizard>
+      <!-- <app-hd-wallet-wizard [resetOnboarding]="resetOnboarding.bind(this)"></app-hd-wallet-wizard> -->
```

Comment:
let's delete instead of commenting unused code

Reply:
Sure it will be done